SConstruct

```diff
@@ -56,7 +56,7 @@ AddOption('--pc-thneed',
 AddOption('--minimal',
           action='store_false',
           dest='extras',
-          default=os.path.islink(Dir('#laika/').abspath),
+          default=True,
```

Comment:
@adeebshihadeh what was this check for?

Reply:
to check if we're on a master or release branch, you can change to rednose